src/containers/AccountModal/forms/Registration.tsx

```diff
@@ -54,10 +54,13 @@ const Registration = () => {
     } catch (error: unknown) {
       if (error instanceof Error) {
         const errorMessage = error.message.toLowerCase();
-        if (errorMessage.includes('customer already exists.')) {
+        if (errorMessage.includes('already exists')) {
```

Comment:
Probably safer to check more verbosely:
```suggestion
        if (errorMessage.includes('customer already exists.') || errorMessage.includes('Account already exists.')) {
```